src/layouts/Container/index.js

```diff
@@ -0,0 +1,234 @@
+
+
+const useStyles = makeStyles({
+    
+// >>>>>>>>>>>>>>>>>>   Container   >>>>>>>>>>>>>>>>
+containerback:{
+    backgroundColor:"white",
+    padding:"1px",
+}, 
+container:{
+    backgroundColor:"white",
+    maxWidth: '1080px',
+    margin:"-60px auto",
+    display: 'flex',
+    justifyContent: 'space-around',
+    padding: '0px',
+    backgroundColor:"white",
+    borderRadius:"20px",
+  
+    '@media screen and (max-width: 850px)': {
+        flexDirection: 'column',
+        alignItems: 'center',
+        margin:"-60px auto",
+    '& > div': {
+    
+      borderRadius: "20px 20px 20px 20px",
+    }
+    },
+    '& > div': {
+        
+        maxWidth: '500px',
+        minWidth: '350px',
+        height: '830px',
+        margin: '2px',
+        width: '100%',
+        '@media screen and (max-width: 1070px)': {
+            minWidth: '280px',
+            height: '930px',
+            maxWidth: '400px',
+        },
+        boxShadow: "0 0 10px rgba(0, 0, 0, 0.3)",
+      border: "0.5px solid brown",
+    },
+    "& > div:nth-child(1)": {
+     textAlign:"center",
+     fontSize:"x-large",
+     borderRadius: "20px 0px 0px 20px",
+     
+     '@media screen and (max-width: 850px)': {
+      borderRadius:"0px 0px",
+      },
+       
+    },"& > div:nth-child(2)": {
+        textAlign:"center",
+        fontSize:"x-large",
+        borderRadius: "0px 0px",
+
+       },"& > div:nth-child(3)": {
+        textAlign:"center",
+        fontSize:"x-large",
+        borderRadius: " 0px 20px 20px 0px",
+        
+        '@media screen and (max-width: 850px)': {
+          borderRadius:"0px 0px",
+          },
+       },
+       '& li':{
+    textDecoration:"none",
+    listStyle:"none",
+       },
+       
+   },
+   designer_img: {
+    width: "25%",
+    height:"10%",
+    borderRadius: '50%',
+    objectFit: 'cover',
+    marginRight: 20,
+  },
+  frontend_img: {
+    width: "23%",
+    height:"10%",
+    borderRadius: '50%',
+    objectFit: 'cover',
+    marginRight: 20,
+  },
+  img_mentor: {
+    width: "30%",
+    height:"10%",
+    borderRadius: '50%',
+    objectFit: 'cover',
+    marginBottom: -10,
+  },
+
+designer_things:{
+    fontSize:16,
+    '& p':{
+      padding:"10px 10px",
+      fontSize:18,
+      margin:"22px",
+    },
+    '& ul':{
+        padding:"0px",
+        listStyle:"none",
+     },
+     
+    textDecoration:"none",
+    listStyle:"none",
+},
+frontend_div2:{
+    fontSize:16,
+  '& p':{
+    fontSize:18,
+    padding:"10px 10px",
+    margin:"20px",
+  },
+  '& ul':{
+    padding:"0px",
+    listStyle:"none",
+ },
+},
+mentor_div2:{
+    fontSize:16,
+  '& p':{
+    fontSize:18,
+    padding:"10px 10px",
+    margin:"22px",
+  },
+  '& ul':{
+    padding:"0px",
+    listStyle:"none",
+ },
+},
+
+});
+
+function Containerbox() {
+  const classes = useStyles();
+
+  return (
+    <div style={{backgroundColor:"white"}}><><><div>
+      <Head>
+        <title>Home-Hashtag404</title>
```

Comment:
please use heading tag

Reply:
did't get that point .can you show me how?